guides/06_client-libraries/01_getting-started-with-the-python-client.md

```diff
@@ -0,0 +1,215 @@
+# Getting Started with the Gradio Python client 
+
+Tags: CLIENT, API, SPACES
+
+
+The Gradio Python client makes it very easy to use any Gradio app as an API. As an example, consider this [Hugging Face Space that transcribes audio files](https://huggingface.co/spaces/abidlabs/whisper) that are recorded from the microphone.
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/gradio-guides/whisper-screenshot.jpg)
+
+Using the `gradio_client` library, we can easily use the Gradio as an API to transcribe audio files programmatically.
+
+Here's the entire code to do it:
+
+```python
+from gradio_client import Client
+
+client = Client("abidlabs/whisper") 
+client.predict("audio_sample.wav")  
+
+>> "This is a test of the whisper speech recognition model."
+```
+
+The Gradio client works with any hosted Gradio app, whether it be an image generator, a text summarizer, a stateful chatbot, a tax calculator, or anything else! The Gradio Client is mostly used with apps hosted on [Hugging Face Spaces](https://hf.space), but your app can be hosted anywhere, such as your own server.
+
+**Prequisites**: To use the Gradio client, you do *not* need to know the `gradio` library in great detail. However, it is helpful to have general familiarity with Gradio's concepts of input and output components.
+
+## Installation
+
+If you already have a recent version of `gradio`, then the `gradio_client` is included as a dependency. 
+
+Otherwise, the lightweight `gradio_client` package can be installed from pip (or pip3) and is tested to work with Python versions 3.9 or higher:
+
+```bash
+$ pip install gradio_client
+```
+
+## Usage
+
+### Connecting to a running Gradio App
```

Comment:
right now only the headers with ## are showing on side nav

<img width="245" alt="Screen Shot 2023-04-08 at 11 24 55 PM" src="https://user-images.githubusercontent.com/9021060/230758040-d25ee909-04b3-4452-af9a-9b5725d8f4b8.png">

Better to show all these headers

<img width="238" alt="Screen Shot 2023-04-08 at 11 26 54 PM" src="https://user-images.githubusercontent.com/9021060/230758077-8d4b0790-6f8b-4fa5-b355-212b48a948cf.png"> 

also some headers are just bold and not headers like 

> `**Connecting a general Gradio app**`

I suggest making all of these ## headers